lib/rspec/rails/example/job_example_group.rb

```diff
@@ -1,23 +1,9 @@
 module RSpec
   module Rails
-    # @api public
-    # Container module for job spec functionality. It is only available if
-    # ActiveJob has been loaded before it.
```

Comment:
Are we sure we want to remove those comments?

Reply:
@benoittgt The diff is misleading, we removed empty modules, not comments only.

Reply:
I think @benoittgt meant that those comments would still be helpful in the description of what's left, i.e.:
```diff
  module RSpec
    module Rails
-     # Container module for job spec functionality.
+     # Container module for job spec functionality. It is only available if
+     # ActiveJob has been loaded before it.
```

Reply:
Yes sorry @knapo. @pirj explained it properly.